gnoi/os/server.go

```diff
@@ -51,3 +59,84 @@ func (s *Server) Activate(ctx context.Context, request *pb.ActivateRequest) (*pb
 	}
 	return &pb.ActivateResponse{Response: &pb.ActivateResponse_ActivateOk{}}, nil
 }
+
+// Install receives an OS package, validates the package and then installs the package.
+func (s *Server) Install(stream pb.OS_InstallServer) error {
+	var resp *pb.InstallRequest
+	var err error
+	if resp, err = stream.Recv(); err != nil {
```

Comment:
Worth logging the sends and receives that are not InstallRequest->bytes with the utils LogProto

Reply:
Sounds good, will do